### ISSUE TITLE
function sum-1 fails for flat-tensors

### ISSUE THREAD

**Issue by dougfort** [themetaschemer/malt#9]
> $ git log -1
> commit 9cd873cc82fe8e4e6eb0a5790251ad360f70ba0f (HEAD -> main, origin/main, origin/HEAD)
> Author: TheMetaSchemer <anurag@mendhekar.com>
> Date:   Sat Feb 25 05:00:03 2023 +0530
> 
>     Fixes reported issue #1.
> 

This shows up in Chapter 5, p 106

```
$ make clean
find . -name 'compiled' | xargs -I% rm -rf %
$ make
Building ...
Tensor implementation: flat-tensors
Tensor implementation (no-duals): flat-tensors
Tensor implementation (no-overrides): flat-tensors
Tensor implementation (no-duals, no-overrides): flat-tensors
Running tests ...
571 tests passed
$ racket
Welcome to Racket v8.8 [cs].
> (require malt)
> (sum-1 (tensor 1.0 2.0 3.0))
sum-1-ρ: arity mismatch;
 the expected number of arguments does not match the given number
  expected: 6
  given: 1
 [,bt for context]
> ,bt
sum-1-ρ: arity mismatch;
 the expected number of arguments does not match the given number
  expected: 6
  given: 1
  context...:
   /home/dougfort/malt/flat-tensors/autodiff/B-prims.rkt:25:2: body of top-level
   /usr/share/racket/pkgs/xrepl-lib/xrepl/xrepl.rkt:1573:0
   /usr/share/racket/collects/racket/repl.rkt:11:26
> 


```

```
$ make clean
find . -name 'compiled' | xargs -I% rm -rf %
$ make
Building ...
Tensor implementation: learner
Tensor implementation (no-duals): learner
Tensor implementation (no-overrides): learner
Tensor implementation (no-duals, no-overrides): learner
Running tests ...
571 tests passed
$ racket
Welcome to Racket v8.8 [cs].
> (require malt)
> (sum-1 (tensor 1.0 2.0 3.0))
6.0

```

**Comment by themetaschemer** [comment]
Thanks for reporting. Investigating.

**Comment by themetaschemer** [comment]
Fixed in 9321e40. 

Please pull.

**Comment by themetaschemer** [comment]
Pushed update 97863c5 to fix comment.
